lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -176,6 +176,13 @@ def initialize(options)
       @devices_list.flatten!
       @devices_list.compact!
       @device_infos = load_devices
+      process_device_options
+    end
+
+    def process_device_options
+      @device_infos.each do |device_info|
+        @pluggable_memory_gb += device_info.pluggable_memory_gb
+      end
```

Comment:
I prefer this to be merged into `memory_replacement_map` to remove the instance variable.

Reply:
We discussed previously, that `pluggable_memory_gb` should be a separate value and autohck should calculate max memory size